assets/js/components/Work/work.query.js

```diff
@@ -118,3 +118,22 @@ export const ADD_WORK_TO_COLLECTION = gql`
     }
   }
 `;
+
+export const DELETE_WORK = gql`
+  mutation deleteWork($workId: ID!) {
+    deleteWork(workId: $workId) {
+      id
```

Comment:
Just checking if we really need to return all this data beyond `id`?   It might make for more to maintain... Unless Apollo cache needs it to make it's own update, but I don't think it does....   Can you experiment removing the extra data and see what happens?

Reply:
I started with ID initially and realized I need to return more data for the navigation to get back to the ingest sheet.